docs/index.md

```diff
@@ -15,6 +15,29 @@ You can set environment variable `SECRETHUB_CREDENTIAL` or read it from disk usi
 
 Use the navigation to the left to read about the available resources.
 
+## Installation
+
+### Terraform 0.13 and later
+
+To install this provider, copy and paste this code into your Terraform configuration. Then, run `terraform init`.
+
+```hcl
+terraform {
+  required_providers {
+    secrethub = {
+      source = "secrethub/secrethub"
+      version = "1.2.2"
```

Comment:
Do we want to add this static version here? We can also just refer to the `Install Provider` button on the top right of this page, then it's always up-to-date.

Reply:
Alternatively, we could auto-bump it like we do a.o. in the pipeline for our GitHub action: https://github.com/secrethub/actions/blob/v0.2.0/.github/workflows/bump-version.yml

Reply:
Could work, but isn't it just as easy to refer to the `Use Provider` button? It's literally on the same page.

Reply:
I think including it in the page is slightly more user friendly, but I'm okay with referring to the button.